core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -154,4 +155,28 @@ public final long getEventLoopBlockedCheckTimeout() {
     public final void setEventLoopBlockedCheckTimeout(final long eventLoopBlockedCheckTimeout) {
         this.eventLoopBlockedCheckTimeout = eventLoopBlockedCheckTimeout;
     }
+
+    /**
+     * Gets the number of maximal concurrent connections the protocol adapter should accept.
+     * <p>
+     * Default is {@link Long#MAX_VALUE}.
+     * 
+     * @return The maximal number of concurrent connections.
+     */
+    public long getConcurrentConnectionsLimit() {
```

Comment:
can we rename this to *getMaxConcurrentConnections*? We use this naming pattern in other places already ...